.travis.yml

```diff
@@ -18,10 +18,10 @@ env:
 
   - TARGET: py36-mysql-file
   - TARGET: py36-mysql-swift
-  - TARGET: py36-mysql-s3
+  - TARGET: py36-mysql-ceph
```

Comment:
This looks unrelated to this PR :(

Reply:
yes indeed it is.
Ideally, I would like to have all backends enabled. There seems to be a timing(?) issue with CI, and lately, the s3 tests were failing with a chance of > 50%